lib/resources/trending_movies_api_provider.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+import '../constants/constants.dart';
+import '../models/item_model.dart';
+
+class TrendingMoviesApiProvider {
+  final String _endpoint = "/trending/movie/week";
+
+  Future<ItemModel> fetchTrendingMovies() async {
+    final response = await http.get(
+      Uri.parse(
+        Constants.uri + _endpoint,
+      ),
+      headers: {
+        "Content-Type": "application/json",
```

Comment:
This String can be added to on Constant file.

Reply:
Done!